### PR TITLE
docs: adds additional openstack docs

### PR DESCRIPTION
**Pull request by nicholaskuechler**
Notes and docs on using various openstack tools

### CODE REVIEW

File: docs/user-guide/openstack-ironic.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      "size_gb": "MAX"
+    }
+  ]
+}
+```
+
+Apply the raid1 config from above:
+
+``` bash
+openstack --os-cloud=${OS_CLOUD} baremetal node set ${NODE_UUID} --target-raid-config raid1-config.json
```
Comment (skrobul) [inline docs/user-guide/openstack-ironic.md:47]: does the `--os-cloud=${OS_CLOUD}` do anything -  isn't the CLI going to use the env variable if it's already defined?

File: docs/user-guide/openstack-placement.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# OpenStack Placement
+
+## Query Placement usages
+
+Get an openstack token:
+
+``` bash
+TOKEN=`openstack token issue -f json | jq -r '.id'`
```
Comment (skrobul) [inline docs/user-guide/openstack-placement.md:8]:
```suggestion
TOKEN=$(openstack token issue -f value -c id)
```

without `jq` dependency

**Review by skrobul — APPROVED** [review]

File: docs/user-guide/openstack-ironic.md (hunk 1 above)
Comment (nicholaskuechler) [reply docs/user-guide/openstack-ironic.md]: removed

File: docs/user-guide/openstack-placement.md (hunk 2 above)
Comment (nicholaskuechler) [reply docs/user-guide/openstack-placement.md]: updated

**Pull request merged**